pages/story/story_data.js

```diff
@@ -0,0 +1,28 @@
+
+//write functions here to take data and push to story_array
+
+export const story_array = 
+[{
+    num : 1,
```

Comment:
`num` is unused i guess. if unused lets remove as we have index from map function to help with uniqueness list

Reply:
forgot to remove, will do